### DISCUSSION TITLE
关于取消 图表目录 章节之间存在的间距

### DISCUSSION THREAD

**Discussion by LaureatePoet** [HFUTTUG/HFUT_Thesis#45] [Q&A] [answered]
## 检查
- [x] 已经阅读《合肥工业大学研究生学位论文写作规范》或《合肥工业大学本科毕业设计（论文）工作实施细则》
- [x] 已经将模板升级到了最新版
- [x] 已经阅读使用说明文档 `hfutthesis-doc.pdf`
- [x] 已经阅读 [常见问题](https://github.com/HFUTTUG/HFUT_Thesis/wiki/常见问题)

## 描述问题
关于取消 图表目录 章节之间存在的间距

截图：
![LK{`8$29U2$42~9%8G2QUVN](https://user-images.githubusercontent.com/42093310/162573805-b7087aec-3710-45ee-92d1-15896cf0f56b.jpg)

## 解决方案
`hfutthesis` 目录是 根据 ` \pkg{titletoc}` 宏包设置目录内容的格式。

至于图表目录章节之间的间距，如果需要一致，
https://github.com/HFUTTUG/HFUT_Thesis/blob/f3319fa92604348b15d928902b7eca3e59a2806e/main.tex#L38-L43
对`main.tex`做 如下修改
```tex
\tableofcontents
\let\origAddvspace\addvspace
\renewcommand\addvspace[1]{}
\listoffigures
\listoftables
\let\addvspace\origAddvspace
```

**Comment by LaureatePoet** [comment, marked as answer]
解决方案如上。

**Comment by HanZhang1995** [comment]
该解决方案不能完美解决，会造成标题行间距缩小，可改成以下形式
```
\tableofcontents
{
\renewcommand*{\addvspace}[1]{}
\listoffigures
}
{
\renewcommand*{\addvspace}[1]{}
\listoftables
}
```

**Comment by zepinglee** [comment]
这个可以使用 ctex 的 `lofskip` 和 `lotskip` 接口进行控制。

<img width="884" alt="Screenshot 2024-05-26 at 13 27 47" src="https://github.com/HFUTTUG/HFUT_Thesis/assets/12290822/6b60d7b3-91b4-4128-b822-491b05d293b1">
